backend/dataall/modules/dataset_sharing/services/data_sharing_service.py

```diff
@@ -207,7 +258,7 @@ def revoke_share(cls, engine: Engine, share_uri: str):
             log.info(f'Still remaining LF resources shared = {existing_shared_items}')
             if not existing_shared_items and revoked_tables:
                 log.info("Clean up LF remaining resources...")
-                clean_up_tables = processor.delete_shared_database()
+                clean_up_tables = processor.clean_up_share() and processor.delete_shared_database()
```

Comment:
Is there a reason we add `processor.clean_up_share() and...` 

I do not believe we have a `clean_up_share()` method for `ProcessLFCrossAccountShare` or `ProcessLFSameAccountShare` classes

Reply:
`clean_up_share()` was removed in `v2m1m0` - I think we can revert this back to `clean_up_tables=processor.delete_shared_database()`

Reply:
delete_shared_database only deletes shared database when share request is rejected. Clean_up_share does a bunch of other things like:
```
        1) deletes S3 access point for this share in this Dataset S3 Bucket
        2) delete_target_role_access_policy to access the above deleted access point
        3) delete_dataset_bucket_key_policy to remove access to the requester IAM role
```
Hence I have added both here.

Reply:
Yes for folder share - `class ProcessS3AccessPointShare()` 

But the above is for Table Shares and I do not believe `def clean_up_share()` is defined for table shares anymore